_assets/javascripts/vendor.js

```diff
@@ -2,3 +2,4 @@
 //= require throttle-debounce-fn/dist/throttle-debounce-fn.js
 //= require fluidbox
 //= require retinajs/dist/retina.js
+//= require svgxuse/svgxuse.js
```

Comment:
According to the plugin's README, this is only required for IE. Please add it to `javascripts.html` with:

```
<!--[if IE]><script src=""></script><![endif]-->
```